app/redhat_insights/default/data/ui/views/events.xml

```diff
@@ -16,7 +16,7 @@ index=redhatinsights AND bundle = "rhel"
 | eval application = upper(substr(application,1,1)).substr(application,2)
 | eval title=case(event_type=="new-recommendation",'payload.rule_description', event_type=="resolved-recommendation",'payload.rule_description', event_type=="deactivated-recommendation", 'payload.rule_description', event_type=="policy-triggered", 'payload.policy_name'." policy triggered", event_type=="drift-baseline-detected","Drift detected from ".'payload.baseline_name'." baseline", event_type=="compliance-below-threshold",'payload.policy_name', event_type=="report-upload-failed", 'payload.error', !isnull(event_type), 'event_type') 
 | eval time=strptime(timestamp, "%Y-%m-%dT%H:%M:%S.%3N")
-| eval url = case(event_type=="drift-baseline-detected", DRIFT_URL.'payload.baseline_id', event_type=="policy-triggered", POLICIES_URL.'payload.policy_id', event_type="compliance-below-threshold", COMPLIANCE_URL.'payload.host_id', !isnull('payload.rule_url'), 'payload.rule_url')
+| eval url = case(event_type=="drift-baseline-detected", DRIFT_URL.'payload.baseline_id', event_type=="report-upload-failed", POLICIES_URL.'payload.request_id', event_type=="policy-triggered", POLICIES_URL.'payload.policy_id', event_type="compliance-below-threshold", COMPLIANCE_URL.'payload.host_id', !isnull('payload.rule_url'), 'payload.rule_url')
```

Comment:
Where does it links to? Is it Payload Tracker?

Reply:
It seems that Payload Tracker has the frontend internal only. Nevertheless, @fellipeh can you please check the generated URL?